README.md

```diff
@@ -3,31 +3,19 @@
 This repository contains the bridging functionality — aptly named Bridget — that will be used as a communication tool between the clients (usually web deliveries) & the Forward Publishing mobile app's webview implementation.
 
 # Why a bridge
- If a client from our Forward Publishing suite requires a mobile app. We offer a native solution to their publishing platform, i.e the web delivery. 
+ If a client from our Forward Publishing suite requires a mobile app, we offer a native solution to their publishing platform, i.e their web delivery. 
  
- Although it mirrors the web delivery in content and simply displays it in a mobile friendly webview, some functionality needs to be communicated from the web world to the mobile native world. Linking or navigation in the web does not behave the same way in the mobile app, simply changing a URL in a webview will simply not do, as it destroys the native immersion and context of what's happenin within the app is lost. 
+ Although it mirrors the web delivery 1:1 in terms of content and displays it in a mobile friendly webview, some functionality needs to be communicated from the web world to the mobile native world. Linking or navigation in the web does not behave the same way in the mobile app, simply changing a URL in a webview will not do, as it destroys the native immersion and context of what's happening within the app is lost. 
 
- That's where Bridget comes in; we can think of it as a thin communicational layer between the web delivery and the mobile app, where functionality in the app is being invoked by message passing from the bridge, and vice versa.
-# Changeslog
+ That's where Bridget comes in; we can think of it as a thin communicational layer between the web delivery and the mobile app, where functionality in the app is being invoked by message passing (think of Android intents) from the bridge, and vice versa.
 
-See the [change log](./CHANGELOG.md)
-
-# Installation
-
-## Node version
-
-The tool works with node 14.x so make sure you have that
-installed first.
- On a Mac with [Homebrew](https://brew.sh) do:
-
-```
-brew install node@14
-```
-
-Otherwise see [the official guide](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+# [Integration Guide](#integration-guide)
+The basis of this document is to explain the process of integrating `bridget` into an existing web delivery with the sole purpose of adding bridging support to the website when browsed through the context of a mobile app via a WebView.
 
+## [Prerequisites](#prerequisites)
```

Comment:
Is this needed as part of the integration guide?

I think we only need to tell about importing the script (as part of the list of steps that are to be performed). I would try to make it very streamlined, i.e. "You need to do this, then this, then this, and this is how you test it in the end."

Reply:
Not as part of the integration guide, no. But, what I would like is to also have documentation for what we expect from a delivery to satisfy in order for the bridge to function properly.